tests/test_cli.py

```diff
@@ -1,18 +1,30 @@
-from click.testing import CliRunner
 from advice_animal.cli import main
+from click.testing import CliRunner
+
 
 def test_smoke():
     runner = CliRunner()
     result = runner.invoke(main)
     assert result.exit_code == 0
 
+
 def test_self_test():
     runner = CliRunner()
     result = runner.invoke(main, ["self", "test"])
     assert result.exit_code == 0
 
 
-def test_check():
+def test_self_check():
     runner = CliRunner()
     result = runner.invoke(main, ["check", "."])
     assert result.exit_code == 0
+
+
+def test_list():
+    runner = CliRunner()
+    result = runner.invoke(main, ["list"])
+    assert (
+        "click_version_option : GREEN \nruff_show_source : YELLOW (preview)\nuse_py_typed : GREEN \n"
```

Comment:
Probably more stable would be use the testing advice dir, which currently only has one called "shouty".  Feel free to add ones called "green", "yellow", "red", and "preview" there too, they don't even really need to perform an action.

See existing use at https://github.com/advice-animal/advice-animal/blob/8252255b168f9b5ad782862f63067a987f7c0397/tests/test_runner.py#L10 (it's just override ADVICE_DIR to tests/advice if you're in the cli)

Reply:
I've updated the tests. I've also changed the output format.